testutil/src/main/java/org/spine3/test/CommandTest.java

```diff
@@ -80,6 +80,33 @@ protected Command createCommand(C commandMessage) {
     }
 
     /**
+     * Creates a command with the passed message.
+     *
+     * <p>Use this method for creating commands of types different than
+     * one which is the subject of the test suite (defined by the generic type {@code <C>}.
+     *
+     * @param commandMessage the message of the command to create
+     * @return new command instance
+     */
+    protected Command createAnotherCommand(Message commandMessage) {
```

Comment:
I think it should be `createDifferentCommand`.

`createAnotherCommand` reads more like `create one more command, same as yesterday, same as day before`.